packages/firestore/lite/src/api/database.ts

```diff
@@ -28,20 +26,30 @@ import {
   CredentialsProvider,
   FirebaseCredentialsProvider
 } from '../../../src/api/credentials';
-import { cast } from './util';
 import { removeComponents } from './components';
 
+declare module '@firebase/component' {
```

Comment:
This is not my area of expertise... this piece of code extends the available services that we can theoretically fetch via `getComponent` to include firestore/lite. This is needed to avoid a compile time error in this file (at the bottom, we have `getFirestore()` for the Lite SDK, which is compiled but not used when building firestore-exp).